google/cloud/storage/internal/raw_client.cc

```diff
@@ -18,13 +18,7 @@
 namespace google {
 namespace cloud {
 namespace storage {
-inline namespace STORAGE_CLIENT_NS {
-std::shared_ptr<internal::RawClient> CreateDefaultClient(
-    ClientOptions options) {
-  return std::make_shared<storage::internal::DefaultClient<>>(
-      std::move(options));
-}
-}  // namespace STORAGE_CLIENT_NS
+inline namespace STORAGE_CLIENT_NS {}  // namespace STORAGE_CLIENT_NS
```

Comment:
This namespace nesting doesn't contain anything anymore... is there a reason it still needs to be here?

Reply:
Yeah, I should have removed the file, there is nothing in it now.  This is done.